mne/channels/channels.py

```diff
@@ -560,11 +561,12 @@ def set_meas_date(self, meas_date):
 
         .. versionadded:: 0.20
         """
-        from ..annotations import _handle_meas_date
-        meas_date = _handle_meas_date(meas_date)
+        from ..annotations import _handle_meas_date_nonUTC
+        meas_date, utc_offset, utc_as_dt = _handle_meas_date_nonUTC(meas_date)
         with self.info._unlock():
             self.info['meas_date'] = meas_date
-
+            if utc_offset is not None:
+                self.info['utc_offset'] = utc_offset  # or utc_offset_dt
```

Comment:
If I'm reading this PR correctly, this is the only place where the new additional return values are used.  Since you seem to only need one of them, why does the function return both?